internal/migrate/is_staged.go

```diff
@@ -0,0 +1,82 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package migrate
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/onflow/cadence"
+	"github.com/onflow/contract-updater/lib/go/templates"
+	"github.com/onflow/flowkit"
+	"github.com/onflow/flowkit/output"
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/internal/scripts"
+)
+
+var isStagedflags struct{}
+
+var IsStagedCommand = &command.Command{
+	Cmd: &cobra.Command{
+		Use:     "is-staged <CONTRACT_NAME>",
+		Short:   "checks to see if the contract is staged for migration",
+		Example: `flow migrate is-staged HelloWorld`,
+		Args:    cobra.MinimumNArgs(2),
+	},
+	Flags: &isStagedflags,
+	RunS:  isStaged,
+}
+
+func isStaged(
+	args []string,
+	globalFlags command.GlobalFlags,
+	_ output.Logger,
+	flow flowkit.Services,
+	state *flowkit.State,
+) (command.Result, error) {
+	contractName := args[0]
+
+	addr, err := getAddressByContractName(state, contractName, globalFlags.Network)
+	if err != nil {
+		return nil, fmt.Errorf("error getting account by contract name: %w", err)
```

Comment:
Getting issues failing here when running:

```
❯ ./main migrate is-staged ExampleNFT --network testnet
Error: requires at least 2 arg(s), only received 1
Usage:
  flow migrate is-staged <CONTRACT_NAME> [flags]
```

With [this flow.json](https://github.com/sisyphusSmiling/cadence-migration-example/blob/cadence-1/flow.json)

Reply:
sorry, found the bug, fixed!